shared/src/commonMain/kotlin/com/vml/tutorial/plantshop/plants/data/FavoritePlantsDataSource.kt

```diff
@@ -0,0 +1,24 @@
+package com.vml.tutorial.plantshop.plants.data
+
+import com.vml.tutorial.plantshop.core.data.toPlant
+import com.vml.tutorial.plantshop.plants.domain.FavoritesDataSource
+import com.vml.tutorial.plantshop.plants.domain.Plant
+import kotlinx.coroutines.async
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.map
+import kotlinx.coroutines.supervisorScope
+
+class FavoritePlantsDataSource : FavoritesDataSource {
```

Comment:
I think this is an extra not necessary class :) You can do all these in PlantsRepository. If you don't mind, can you put it there?

Reply:
Moved to PlantsRepository